src/XmlUtility.ps1

```diff
@@ -0,0 +1,128 @@
+﻿$WikiConfig = @{
```

Comment:
You should't use the `Wiki` keyword here. This file is not associated to a Wiki.

Reply:
Fixed